eng/versioning/version_client.txt

```diff
@@ -67,20 +67,20 @@ com.azure:azure-aot-graalvm-support-netty;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-aot-graalvm-samples;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-aot-graalvm-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-code-customization-parent;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-callautomation;1.1.3;1.2.0-beta.1
+com.azure:azure-communication-callautomation;1.1.4;1.2.0-beta.1
 com.azure:azure-communication-callingserver;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-communication-chat;1.5.0;1.6.0-beta.1
-com.azure:azure-communication-common;1.3.1;1.4.0-beta.1
+com.azure:azure-communication-common;1.3.2;1.4.0-beta.1
 com.azure:azure-communication-common-perf;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-communication-email;1.0.11;1.1.0-beta.1
-com.azure:azure-communication-identity;1.5.3;1.6.0-beta.1
-com.azure:azure-communication-jobrouter;1.1.2;2.0.0-beta.1
-com.azure:azure-communication-messages;1.0.1;1.1.0-beta.1
+com.azure:azure-communication-email;1.0.12;1.1.0-beta.1
+com.azure:azure-communication-identity;1.5.4;1.6.0-beta.1
+com.azure:azure-communication-jobrouter;1.1.3;2.0.0-beta.1
+com.azure:azure-communication-messages;1.0.2;1.1.0-beta.1
 com.azure:azure-communication-networktraversal;1.1.0;1.2.0-beta.1
-com.azure:azure-communication-phonenumbers;1.1.11;1.2.0-beta.4
-com.azure:azure-communication-rooms;1.0.9;1.1.0
-com.azure:azure-communication-sms;1.1.22;1.2.0-beta.1
-com.azure:azure-containers-containerregistry;1.2.6;1.3.0-beta.1
+com.azure:azure-communication-phonenumbers;1.1.12;1.2.0-beta.4
+com.azure:azure-communication-rooms;1.1.1;1.2.0-beta.1
```

Comment:
[1.1.0 ](https://mvnrepository.com/artifact/com.azure/azure-communication-rooms/1.1.0)was released earlier of this month but still use the previous version of azure core and communication core.

Reply:
And the version incremental PR is not merge back to main yet:
https://github.com/Azure/azure-sdk-for-java/pull/39778/files